### ISSUE TITLE
Outdated examples

### ISSUE THREAD

**Issue by PaulDotSH** [weiznich/diesel_async#84]
The examples provided in the README and on crates.io are outdated and do not work.
Example
The code snipped
```rs
let mut connection = AsyncPgConnection::establish(std::env::var("DATABASE_URL")?).await?;

// use ordinary diesel query dsl to construct your query
let data: Vec<User> = users::table
    .filter(users::id.gt(0))
    .or_filter(users::name.like("%Luke"))
    .select(User::as_select())
    // execute the query via the provided
    // async `diesel_async::RunQueryDsl`
    .load(&mut connection)
    .await?;
```
Will result in the following error

```
error[E0277]: the trait bound `User: FromSqlRow<_, Pg>` is not satisfied
   --> src/main.rs:62:15
    |
62  |         .load(&mut connection)
    |          ---- ^^^^^^^^^^^^^^^ the trait `FromSqlRow<_, Pg>` is not implemented for `User`
    |          |
    |          required by a bound introduced by this call

```

Also the bb8 code snippet
```rs
    let config = AsyncDieselConnectionManager::<diesel_async::AsyncPgConnection>::new(std::env::var("DATABASE_URL")?);
    let pool = Pool::builder().build(config).await?;

// checkout a connection from the pool
    let mut conn = pool.get().await?;

// use the connection as ordinary diesel-async connection
    let res = users::table.select(User::as_select()).load::(&mut conn).await?;
```
Will result in the following error
```
error: field expressions cannot have generic arguments
  --> src/main.rs:61:54
   |
61 |     let res = users::table.select(User::as_select()).load::(&mut conn).await?;
   |                                                      ^^^^^^^^^^^^^^^^^

```

And even if we use the turbofish syntax ``load::<User>`` the code will still output errors
```
   --> src/main.rs:61:67
    |
61  |     let res = users::table.select(User::as_select()).load::<User>(&mut conn).await?;
    |                                                      ----         ^^^^^^^^^ the trait `FromSqlRow<_, _>` is not implemented for `User`
    |                                                      |
    |                                                      required by a bound introduced by this call
    |
```

**Comment by weiznich** [comment]
I've fixed some minor things in the readme in https://github.com/weiznich/diesel_async/commit/9e3d31e5b02108811bd4ff3a538bfa00ce1d8837. I also verified that the code now compiles without errors. None of them should have caused that error, so I'm unsure whatever was going on for you. 

I'm closing this issue as cannot reproduce. Feel free to come back with an reproducible example of your problem.

**Comment by PaulDotSH** [comment]
> I've fixed some minor things in the readme in [9e3d31e](https://github.com/weiznich/diesel_async/commit/9e3d31e5b02108811bd4ff3a538bfa00ce1d8837). I also verified that the code now compiles without errors. None of them should have caused that error, so I'm unsure whatever was going on for you.
> 
> I'm closing this issue as cannot reproduce. Feel free to come back with an reproducible example of your problem.

Sorry, I wasn't on my computer when I saw your response, it was a problem on my end, the example work fine, sorry

**Comment by weiznich** [comment]
No worries. Thanks for coming back and confirming that the examples work.
